connector/mqtt/vernemq/broker/src/dojot_acl_plugin/readme.md

```diff
@@ -1,38 +1,51 @@
 # Dojot ACL Plugin
```

Comment:
Can we add a link to vernemq's documentation, showing how to develop a plugin? just for a reference.